.eslintignore

```diff
@@ -1,3 +1,4 @@
 /js-cardano-wasm
 /dll
-/chrome/3rd-party/trezor
\ No newline at end of file
+/chrome/3rd-party/trezor
+app/api/ada/hardwareWallet/utils.js
```

Comment:
since, this file is replaced by ledger bridge package so now this line can be deleted.